### DISCUSSION TITLE
Not Syncing

### DISCUSSION THREAD

**Discussion by jatsc** [Chia-Network/chia-blockchain#8841] [Support]
I restarted the computer and noy I can not sync
It stops in 
Syncing 994,998 / 1,005,035

**Comment by BrandtH22** [comment]
@jatsc , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
